2022-02-07 19:21:08,599 INFO    MainThread:20319 [internal.py:wandb_internal():91] W&B internal server running at pid: 20319, started at: 2022-02-07 19:21:08.599608
2022-02-07 19:21:08,602 DEBUG   HandlerThread:20319 [handler.py:handle_request():124] handle_request: run_start
2022-02-07 19:21:08,605 INFO    WriterThread:20319 [datastore.py:open_for_write():80] open: output/wandb/offline-run-20220207_192107-b983av9g/run-b983av9g.wandb
2022-02-07 19:21:10,166 DEBUG   HandlerThread:20319 [meta.py:__init__():39] meta init
2022-02-07 19:21:10,166 DEBUG   HandlerThread:20319 [meta.py:__init__():53] meta init done
2022-02-07 19:21:10,166 DEBUG   HandlerThread:20319 [meta.py:probe():210] probe
2022-02-07 19:21:10,173 DEBUG   HandlerThread:20319 [meta.py:_setup_git():200] setup git
2022-02-07 19:21:10,204 DEBUG   HandlerThread:20319 [meta.py:_setup_git():207] setup git done
2022-02-07 19:21:10,204 DEBUG   HandlerThread:20319 [meta.py:_save_pip():57] save pip
2022-02-07 19:21:10,205 DEBUG   HandlerThread:20319 [meta.py:_save_pip():71] save pip done
2022-02-07 19:21:10,206 DEBUG   HandlerThread:20319 [meta.py:_save_conda():78] save conda
2022-02-07 19:21:13,576 DEBUG   HandlerThread:20319 [meta.py:_save_conda():86] save conda done
2022-02-07 19:21:13,576 DEBUG   HandlerThread:20319 [meta.py:probe():252] probe done
2022-02-07 19:23:33,691 WARNING MainThread:20319 [internal.py:wandb_internal():147] Internal process interrupt: 1
2022-02-07 19:23:33,884 WARNING MainThread:20319 [internal.py:wandb_internal():147] Internal process interrupt: 2
2022-02-07 19:23:33,884 ERROR   MainThread:20319 [internal.py:wandb_internal():150] Internal process interrupted.
2022-02-07 19:23:34,210 INFO    WriterThread:20319 [datastore.py:close():288] close: output/wandb/offline-run-20220207_192107-b983av9g/run-b983av9g.wandb
2022-02-07 19:23:34,640 INFO    SenderThread:20319 [sender.py:finish():947] shutting down sender
2022-02-07 19:23:34,640 INFO    MainThread:20319 [internal.py:handle_exit():78] Internal process exited
